docs/3.api/5.kit/14.builder.md

```diff
@@ -112,7 +112,7 @@ Extends the Vite configuration. Callback function can be called multiple times,
 ### Type
 
 ```ts
-function extendViteConfig (callback: ((config: ViteConfig) => void), options?: ExtendViteConfigOptions): void
+function extendViteConfig (callback: ((config: UserConfig) => void), options?: ExtendViteConfigOptions): void
```

Comment:
Although technically correct (as in, it's the `UserConfig` export from `vite`), this is less intuitive for users who need to know that this configuration is _vite_ configuration. Can you think of a better way of communicating it?